### ISSUE TITLE
Custom command result dialog doesn't show stderr output

### ISSUE THREAD

**Issue by joaosigma** [ForkIssues/TrackerWin#1751]
I have a custom command whose final status window doesn't show any output at all. And the command outputed useful information (e.g.: a link to follow next).

![fork1](https://user-images.githubusercontent.com/597285/212196499-366f2962-4dc0-4247-8cb1-1837395d5e8e.png)

When the output should have been:
![fork2](https://user-images.githubusercontent.com/597285/212196504-e2157e90-a545-45b9-bd18-3b51f72be283.png)

This happened in Windows. In macOs the output in shown on the command window.

**Comment by DanPristupov** [comment]
> I have a custom command whose final status window doesn't show any output at all.

I need to see the custom command

Since we are here I'd like to mention that starting `Fork 1.79` Push dialog remembers last used custom refspec, which can be useful for gerrit.

**Comment by joaosigma** [comment]
Here's the custom command:
```json
{
    "name": "Gerrit",
    "target": "ref",
    "refTargets": [
      "localbranch"
    ],
    "action": {
      "type": "process",
      "path": "$git",
      "args": "push gerrit HEAD:refs/for/xxxx",
      "showOutput": true,
      "waitForExit": true
    }
  }
```

**Comment by DanPristupov** [comment]
It turned out that `push` has empty output on success. The useful information you mentioned was in stderr.

Anyway, I updated the output to show both stdout and stderr.

**Comment by DanPristupov** [comment]
We just released Fork 1.81 with the update.

**Comment by joaosigma** [comment]
I can confirm the issue is fixed on 1.81 with the example I gave, however, it still happens for this command:
```json
{
    "name": "Generate traces",
    "target": "ref",
    "refTargets": [
      "localbranch"
    ],
    "ui": {
      "title": "Generate traces",
      "description": "Make sure that the branch '$sha' has been uploaded.\\nProceed with trace generation?",
      "buttons": [
        {
          "title": "OK",
          "action": {
            "type": "sh",
            "script": "python ...",
            "showOutput": true,
            "waitForExit": true
          }
        },
        {
          "title": "Cancel",
          "action": {
            "type": "cancel"
          }
        }
      ]
    }
  }
```

Other small details:
 - the "description" field doesn't recognize `\n` (or `\r\n`) which would be nice.
 - in the original fixed issue, links in the output message box aren't recognized (aren't clickable like in Mac), which would also be nice.

**Comment by DanPristupov** [comment]
> the "description" field doesn't recognize \n (or \r\n) which would be nice.

The description field doesn't recognize `\n`, however json does. For example: `"description": "Make sure that the branch '$sha' has been uploaded.\nProceed with trace generation?",` works.

**Comment by DanPristupov** [comment]
We just released 1.84 with a few related improvements.
